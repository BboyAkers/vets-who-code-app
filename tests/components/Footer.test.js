import React from 'react'
import Footer from '../../src/components/Footer'
import { render } from 'react-testing-library'

describe('<Footer />', () => {
  let wrapper

  beforeEach(() => wrapper = render(<Footer />))

  test('should render correctly', () => {
    const { container } = wrapper;
    expect(container.firstChild).toMatchSnapshot()
  })

})
