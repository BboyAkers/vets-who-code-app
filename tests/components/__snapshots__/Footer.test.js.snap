// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render correctly 1`] = `
<div>
  <section
    className="footer-widgets pad-extra"
  >
    <div
      className="container"
    >
      <div
        className="row"
      >
        <div
          className="col-md-4"
        >
          <div
            className="widgets-content footer-widget-wrap"
          >
            <h3
              className="widgets-title"
            >
              About #VetsWhoCode
            </h3>
            <p>
              FRAGO, doing business as #VetsWhoCode, is an exempt organization as described in Section 501(c)(3) of the Internal Revenue Code. Our EIN is 47-3555231.
            </p>
            <div
              className="footer-social"
            >
              <a
                href="http://bit.ly/vetswhocode-fb"
              >
                <i
                  className="fa fa-facebook-f"
                />
              </a>
              <a
                href="http://bit.ly/vets-who-code-twitter"
              >
                <i
                  className="fa fa-twitter"
                />
              </a>
              <a
                href="http://bit.ly/2omsjdX"
              >
                <i
                  className="fa fa-codepen"
                />
              </a>
              <a
                href="https://bit.ly/2sJ6W9f"
              >
                <i
                  className="fa fa-instagram"
                />
              </a>
              <a
                href="https://bit.ly/2sFKi1u"
              >
                <i
                  className="fa fa-linkedin"
                />
              </a>
              <a
                href="http://bit.ly/vwc-github"
              >
                <i
                  className="fa fa-github"
                />
              </a>
            </div>
          </div>
        </div>
        <div
          className="col-md-4"
        >
          <div
            className="footer-widget-wrap"
          >
            <div
              className="row"
            >
              <div
                className="col-md-11 col-md-offset-1"
              >
                <h3
                  className="widgets-title"
                >
                  Site Map
                </h3>
              </div>
              <div
                className="col-md-5 col-md-offset-1"
              >
                <div
                  className="widgets-content"
                >
                  <ul
                    className="widgets-list"
                  >
                    <li>
                      <GatsbyLink
                        to="/"
                      >
                        Home
                      </GatsbyLink>
                    </li>
                    <li>
                      <GatsbyLink
                        to="/about"
                      >
                        About
                      </GatsbyLink>
                    </li>
                    <li>
                      <GatsbyLink
                        to="/syllabus"
                      >
                        Syllabus
                      </GatsbyLink>
                    </li>
                    <li>
                      <GatsbyLink
                        to="/mentor"
                      >
                        Mentor
                      </GatsbyLink>
                    </li>
                  </ul>
                </div>
              </div>
              <div
                className="col-md-6"
              >
                <div
                  className="widgets-content"
                >
                  <ul
                    className="widgets-list"
                  >
                    <li>
                      <GatsbyLink
                        to="/apply"
                      >
                        Apply
                      </GatsbyLink>
                    </li>
                    <li>
                      <GatsbyLink
                        to="/donate"
                      >
                        Donate
                      </GatsbyLink>
                    </li>
                    <li>
                      <GatsbyLink
                        to="/contact"
                      >
                        Contact Us
                      </GatsbyLink>
                    </li>
                    <li>
                      <a
                        href="https://medium.com/vets-who-code"
                        rel="noopener noreferrer"
                        target="_blank"
                      >
                        <span>
                          Blog
                        </span>
                      </a>
                    </li>
                  </ul>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          className="col-md-4"
        >
          <div
            className="widgets-content"
          >
            <h3
              className="widgets-title"
            >
              Contact Us
            </h3>
            <div
              className="office map"
            >
              <p>
                <span>
                  Hi, if you have any questions, please drop us a line.
                </span>
                <br />
                Email: hello@vetswhocode.io
                <br />
                Phone: (917) 960-3806
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>
  <footer
    className="section footer"
    id="footer"
  >
    <div
      className="container"
    >
      <div
        className="row"
      >
        <div
          className="col-md-6 col-sm-6"
        >
          <p
            className="copyright"
          >
            <span
              className="fa fa-copyright"
            />
             #VetsWhoCode
          </p>
        </div>
        <div
          className="col-md-6 col-sm-6"
        >
          <ul
            className="footer-menu"
          >
            <li>
              Made with 
              <span
                className="fa fa-heart red"
              />
               By #VetsWhoCode.
            </li>
          </ul>
        </div>
      </div>
    </div>
  </footer>
</div>
`;
